pages/parent/signup.tsx

```diff
@@ -421,55 +401,17 @@ export default function ParticipantInfo({
             body: JSON.stringify(userData),
         };
         const response = await fetch("api/user", request);
```

Comment:
The reason why this is doing 404 is because it's calling `/parent/user/user` on your page. We want do not want to do relative path:
```suggestion
        const response = await fetch("/api/user", request);
```

Similarly, please fix this for the `volunteer/signup` page!

After, I'm getting this error, so we definitely want to use validator's phone validate method for consistency: 
```json
{
"error": "Invalid phone number provided: "
}
```